scripts/BHM-example.jl

```diff
@@ -2,7 +2,11 @@
 
 # This is an example calculation finding the ground state of
 # a 1D Bose-Hubbard chain with 6 particles in 6 lattice sites.
-# The Julia run-able script is in [`scripts/BHM-example.jl`](../../scripts/BHM-example.jl).
+
+# A runnable script for this example is located [here](../../../scripts/BHM-example.jl).
```

Comment:
This link to the actual script (../../../scripts/BHM-example.jl) still does not work. I am not sure whether we can link the scripts under the docs directories. Maybe we can just link them to the actual GitHub repository such as https://github.com/joachimbrand/Rimu.jl/blob/develop/scripts/BHM-example.jl ?

Reply:
Ah yes, perhaps I only tested those links in the local build. I will add the full links